content/docs/talks.md

```diff
@@ -0,0 +1,23 @@
++++
+title="Talks"
+weight=2
+getting-started=true
++++
+
+## Talks
+
+We love talks to share the latest development updates, covering Buildpacks basics and more, receive feedback and questions, and get to know other members of the community.
+
+### Introductory Talks  
+
+If you are new to the project, you can watch some of introductory talks on [Cloud Native Buildpacks' YouTube channel](https://www.youtube.com/@buildpacks)
+
+### Conference Talks
+
+{{< talks >}}
+
+More conference talks are available under the [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.  
```

Comment:
```suggestion
### Conference Talks

Check out some of our most recent and exciting conference talks below. More talks are available in our [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.

{{< talks >}}

If you are interested in giving a talk about buildpacks, the linked slides may provide a useful starting point. Please feel free to reach out in Slack if you'd like input or help from the CNB team!
```

It's a bit weird that the `{{< talks >}}` block also drops in a "Resources" with downloadable slides. It makes the "More conference talks are available..." text seem a bit out of place. WDYT?